Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -864,9 +775,6 @@ static __forceinline u32 TranslatePageAddress(const u32 _Address, const XCheckTL
 
 		u32 pteg_addr = ((hash & PowerPC::ppcState.pagetable_hashmask) << 6) | PowerPC::ppcState.pagetable_base;
 
-		if ((pteg_addr >> 28) == 1)
-			base_mem = Memory::m_pEXRAM;
-
```

Comment:
No, that's not true either. Memory::Base points to a 4GB address space that mirrors the GC/Wii memory layout, but only some parts of it are actually backed by real memory. The rest segfault.